### PR TITLE
active link menu

### PR DESCRIPTION
**Pull request by turtuga**
Fix this 

to

### CODE REVIEW

File: resources/views/menu/admin_menu.blade.php [hunk 1]
```diff
@@ -20,19 +20,30 @@
...
+            $linkAttributes =  'href="#' . str_slug($item->title, '-') .'-dropdown-element" data-toggle="collapse" aria-expanded="'. (in_array('active', $listItemClass) ? 'true' : 'false').'"';
+            array_push($listItemClass, 'dropdown');
+        }
+        else
+        {
             $linkAttributes =  'href="' . url($item->link()) .'"';
         }
 
-        $listItemClass = implode(" ",$listItemClass);
+        //$listItemClass = implode(" ", $listItemClass);
```
Comment (marktopper) [inline resources/views/menu/admin_menu.blade.php]: We should not leave commented out code like this

**Review by marktopper — CHANGES_REQUESTED** [review]

File: resources/views/menu/admin_menu.blade.php (hunk 1 above)
Comment (turtuga) [reply resources/views/menu/admin_menu.blade.php]: sorry changed :)

**Review by marktopper — APPROVED** [review]

**Comment by marktopper** [comment]
Thanks, will test this out.

**Comment by marktopper** [comment]
Everything works great, really great job 🎉

**Comment by akazorg** [comment]
Great job! This was really missing

**Pull request merged**